app/views/declaration/supervising_customs_office.scala.html

```diff
@@ -14,50 +14,78 @@
  * limitations under the License.
  *@
 
-@import controllers.declaration.routes
-@import models.requests.JourneyRequest
-@import forms.declaration.SupervisingCustomsOffice
 @import controllers.navigation.Navigator
+@import forms.declaration.SupervisingCustomsOffice
 @import services.CustomsOffices
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components.gds._
+@import models.requests.JourneyRequest
 @import services.view.AutoCompleteItem
 @import views.Title
+@import views.BackButton
+@import views.components.gds.Styles
+@import scala.collection.immutable
+@import play.twirl.api.HtmlFormat
 
-@this(main_template: views.html.main_template)
-
+@this(
+    govukLayout: gdsMainTemplate,
+    govukButton: GovukButton,
+    govukRadios: GovukRadios,
+    govukDetails : GovukDetails,
+    govukFieldset: GovukFieldset,
+    errorSummary: errorSummary,
+    sectionHeader: sectionHeader,
+    saveButtons: saveButtons,
+    tariffDetails: tariffDetails,
+    formHelper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF
+)
 
 @(mode: Mode, form: Form[SupervisingCustomsOffice])(implicit request: JourneyRequest[_], messages: Messages)
 
-@tariffBody = { @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))}
-
-    @main_template(
-        title = Title("declaration.warehouse.supervisingCustomsOffice.title")
-    ) {
+@pageHint = {
+    <span class="govuk-hint">
+        @messages("declaration.warehouse.supervisingCustomsOffice.hint")
+    </span>
+}
 
-        @helper.form(routes.SupervisingCustomsOfficeController.submit(mode), 'autoComplete -> "off") {
-            @helper.CSRF.formField
+@tariff = {
+    @components.details(messages("site.details.summary_text_these")) {
+        @Html(messages("declaration.type.supervisingCustomsOfficeSummaryText", components.details_content_link()))
+    }
+}
 
-            @components.back_link(Navigator.backLink(SupervisingCustomsOffice, mode)(request))
+@autocomplete = {
+    @components.fields.field_accessible_autocomplete(
+        field = form("supervisingCustomsOffice"),
+        label = "",
```

Comment:
I'm thinking if the empty label cause any accessibility issues.

Reply:
Yes I agree, but the prototype does not have one. I have asked this question to Chris, but he said that the screen reader does not find a problem